PyTorch/LanguageModeling/BERT/data/bertPrep.py

```diff
@@ -127,15 +130,15 @@ def main(args):
     elif args.action == 'create_tfrecord_files':
         assert False, 'TFrecord creation not supported in this PyTorch model example release.' \
                       ''
-        if not os.path.exists(directory_structure['tfrecord']):
```

Comment:
Please make sure the directory is created before the subdirectory

Reply:
os.makedirs can create recursively, should be ok?

Reply:
Right, I was thinking mkdir(). Should be fine